docs/run.bat

```diff
@@ -1,6 +1,6 @@
 del _site /s /q
 
 rem Documentation spell check
-..\build.cmd --target SpellCheck
+pwsh -ExecutionPolicy ByPass -NoProfile -File "..\build.ps1" --target SpellCheck --no-logo
```

Comment:
`pwsh` is not an alias for powershell on my machine

```suggestion
powershell -ExecutionPolicy ByPass -NoProfile -File "..\build.ps1" --target SpellCheck --no-logo
```

Reply:
It seems `pwsh` is powershell 7, I have powershell 5.1

https://techcommunity.microsoft.com/t5/windows-powershell/pwsh-exe-not-recognized-even-after-adding-path/m-p/3680174

Reply:
Using `powershell` works for me as well, so I've updated the PR. Does it work for you too?

Reply:
Yes that works